client/my-sites/plugins/plugins-list/style.scss

```diff
@@ -54,3 +54,23 @@ body.is-section-plugins .plugin-management-wrapper,
         padding-left: 80px;
     }
 }
+
+.sites-manage-plugin-status-button {
+    padding: 0;
+    display: contents;
+    &:focus:not(:disabled) {
+        outline: none;
```

Comment:
This is a no-no. Keyboard navigation needs to be visible. The keyboard focus already appears on the other controls in the table, it just randomly disappears and re-appears when you tab through this button.